pkg/llm-d-inference-sim/utils.go

```diff
@@ -124,3 +126,8 @@ func randomInt(max int, startFromOne bool) int {
 func flipCoin() bool {
 	return randomInt(1, false) != 0
 }
+
+func tokenize(text string) []string {
+	re := regexp.MustCompile(`(\{|\}|:|,|-|\.|\?|\!|;|@|#|\$|%|\^|&|\*|\(|\)|\+|\-|_|~|/|\\|>|<|\[|\]|=|"|\w+)(\s*)`)
```

Comment:
Compile the regular expression once in an init() function. MustCompile panics if the compile fails.